mp4parse/src/lib.rs

```diff
@@ -2394,6 +2414,7 @@ fn read_moov<T: Read>(f: &mut BMFFBox<T>, context: Option<MediaContext>) -> Resu
         mvex,
         psshs,
         userdata,
+        metadata,
```

Comment:
```suggestion
        #[cfg(feature = "meta-xml")]
        metadata,
```
Required based on putting `MediaContext.metadata` behind the feature flag